docs/src/main/sphinx/installation/cli.rst

```diff
@@ -59,16 +59,16 @@ Use ``--help`` to see information about specifying the keystore, truststore, and
 other authentication details as required. If using Kerberos, see :doc:`/security/cli`.
 
 External authentication - SSO
------------------------------
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-The ``--external-authentication`` option is used for browser-based SSO
+Use the ``--external-authentication`` option for browser-based SSO
 authentication, as detailed in :doc:`/security/oauth2`. With this configuration,
 the CLI displays a URL that you must open in a web browser for authentication.
 
 The detailed behavior is as follows:
 
 * Start the CLI with the ``--external-authentication`` option.
-* CLI starts and connects to Trino.
+* The CLI starts and connects to Trino.
```

Comment:
We're actually missing a step here, since we don't connect/authenticate until the user runs a query.

Reply:
Maybe change the first one to 

> Start the CLI ... and execute a query.

Reply:
ok